system/Autoloader/Autoloader.php

```diff
@@ -100,11 +103,11 @@ class Autoloader
 	 * the valid parts that we'll need.
 	 *
 	 * @param \Config\Autoload $config
-	 * @param \Config\Modules  $moduleConfig
+	 * @param \Config\Modules  $modules
```

Comment:
`@param Modules $modules` can be used, no longer require fqcn

Reply:
My preference has always been FQCN for phpdocs since that will help my IDE in resolving names. If others would want short class names, I'll oblige.

Reply:
I have no preferences in that matter. But maybe we should standardize it somehow in the codebase.

Reply:
I prefer to use the shortname for anything that has a `use` statement, but I don't use an IDE so it doesn't actually have negative consequences. I will say that PHPStan accepts the shortname.